lib/taskana-data/src/main/java/pro/taskana/sampledata/SampleDataGenerator.java

```diff
@@ -33,18 +35,18 @@
   private static final String CACHED_DROPDB = "DROPDB";
   private static HashMap<String, List<String>> cachedScripts = new HashMap<>();
   private final DataSource dataSource;
-  private final LocalDateTime now;
+  private final ZonedDateTime now;
   /**
    * This value cannot be automatically obtained by connection.getSchema(), because setting not yet
    * existing schema will result into an SQL Exception.
    */
   private final String schema;
 
   public SampleDataGenerator(DataSource dataSource, String schema) {
-    this(dataSource, schema, LocalDateTime.now());
+    this(dataSource, schema, Instant.now().atZone(ZoneId.of("UTC")));
```

Comment:
ZonedDateTime.now()

Reply:
ZonedDateTime.now() gives me the actual time in the current time zone, not in the UTC time zone. If I invoke ZonedDateTime.now() and print it, I see 2020-01-22 13:53:05, which is the current moment in the local time zone.
If I print at the same time the output of Instant.now().atZone(ZoneId.of("UTC")), I see 2020-01-22 12:53:05 which is the current moment in UTC time zone.

Reply:
okay. Why did you use ZonedDateTime.now() in the SqlReplacerTests then? :)

Reply:
I was a little confused, because the change from LocalDateTime -> ZonedDateTime was not consistent. That's why I am asking :+1:

Reply:
In the SqlReplacerTests, it doesn't matter if you are using local time or utc time. These times are not persisted, only the correct matching and correct relative calculations are checked. However, I can change this in the SqlReplacer, too.

Reply:
wanted to say " ... change this in the SqlReplacerTests, too"

Reply:
If it doesn't matter then I dont care.